2019/2019-11-30-pycon-talk.md

```diff
@@ -0,0 +1,97 @@
+# **Title**
+
+The Traps and Challenges of Machine Learning Outside the Kaggle Cloister
+
+---
+
+_Puns, jokes, or "hooks" in titles are okay, but make sure that if all someone knew was the title, they still would have some idea what the presentation is about._
+
+_Please do not include any personally identifiable information. The initial round of reviews are anonymous, and this field will visible to reviewers._
+
+# **Description**
+
+Machine learning is more art than science, and it isn't alway clear how you go from Kaggle competitions to building your own machine learning project from scratch. You have a cool idea, but how do you turn it into an application that you can show off to your friends, basking in the warmth of their envy? More importantly, what mistakes do you need to avoid in order to keep the hope of such basking alive?
+
+I can't guarantee future success, but come along anyway to learn from my missteps, assumptions exposed and exploded, and the iterative process that makes such losing no disaster. You'll learn about how to deal with haphazard data entry, depending on the kidness of strangers, boxes of different shapes and sizes, and, finally, what to do when all of this changes, because it will.
```

Comment:
`learn from my missteps, assumptions exposed and exploded, and the iterative process that makes such losing no disaster.` 
i get what you're saying here (taking the listener on a journey), but it's a little wordy.

Reply:
Yeah, I knew that last bit was a stretch, but I had to go for it

Reply:
@alexrogers you are wrong, that's craig style at its purest!